app/components/publish/providers/school_placements/explainer_component.html.erb

```diff
@@ -0,0 +1,16 @@
+<p class="govuk-body">
+  Add the schools you can offer placements in. A placement school is where candidates will go to get classroom
+  experience.
+</p>
+
+<div class="govuk-inset-text">
+  <p class="govuk-body">
+    Your courses will not appear in candidate's location searches if you do not add placement schools to them.
+  </p>
+
+  <%= link_to "Find out more about why you should add school placement locations", t("links.how_to.add_school_placements") %>
```

Comment:
```suggestion
  <%= link_to "Find out more about why you should add school placement locations", t("links.how_to.add_schools_and_study_sites") %>
```

Reply:
I deliberately made this `add_school_placements` as this is what makes sense for the view. In all honesty I thought there was a link for the other page. 
Even though the link goes to information about both, I prefer the key being specific to the usage i.e. only about schools

Reply:
I think that makes sense, but also I like Avin's suggestion because it is a consistent way of naming links. Both parties make good points.

Reply:
That means if we have to link to guidance about study sites, we would have to create a new translation with the same link. 

It feels wrong to put the link in the generic link section of the translation file but restrict its use in that way. 

Anyways this is very minor, i'm happy to approve.

Reply:
This should be `govuk_link_to` btw some of the formatting is off